src/main/scala/io/iohk/ethereum/transactions/PendingTransactionsManager.scala

```diff
@@ -0,0 +1,89 @@
+package io.iohk.ethereum.transactions
+
+import akka.actor.{Actor, ActorRef, Props}
+import akka.util.{ByteString, Timeout}
+import io.iohk.ethereum.domain.SignedTransaction
+import io.iohk.ethereum.network.PeerManagerActor.Peers
+import io.iohk.ethereum.network.{Peer, PeerActor, PeerId, PeerManagerActor}
+import io.iohk.ethereum.network.PeerMessageBusActor.{MessageClassifier, MessageFromPeer, PeerSelector, Subscribe}
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.SignedTransactions
+
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object PendingTransactionsManager {
+  def props(peerManager: ActorRef, peerMessageBus: ActorRef): Props =
+    Props(new PendingTransactionsManager(peerManager, peerMessageBus))
+
+  case class AddTransaction(signedTransaction: SignedTransaction)
+  private case class NotifyPeer(signedTransactions: Seq[SignedTransaction], peer: Peer)
```

Comment:
Not really. It's send from inside the future (where I can't update the var directly)